VMAccess/vmaccess.py

```diff
@@ -65,9 +65,12 @@ def enable():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName)
     hutil.do_parse_context('Enable')
     try:
+	public_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('publicSettings')
         protect_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('protectedSettings')
         reset_ssh = protect_settings.get('reset_ssh')
-        # check port each time the VM boots up
+        check_disk = public_settings.get('check_disk')
+	repair_disk = public_settings.get('repair_disk')
```

Comment:
Hi, @Snehsa

Would you please expand "tab" to 4 spaces at the beginning of the line? Using "tab" could introduce issue since different text editors would interpreter it differently.